.gitignore

```diff
@@ -40,3 +40,4 @@ jspm_packages
 # generated files
 lib
 .tmp
+test/bundle.js
```

Comment:
No new line EOF. This is needed because of Git diffs. You should set this as a standard so you need not to think about it.